docs/JSON-RPC.md

```diff
@@ -129,6 +129,23 @@ Results:
 | result.version | string | The Jamulus version. |
 
 
+### jamulus/pollServerList
+
+Request list of servers in a directory
+
+Parameters:
+
+| Name | Type | Description |
+| --- | --- | --- |
+| params.directory | string | socket address of directory to query, e.g. anygenre1.jamulus.io:22124. |
```

Comment:
```suggestion
| params.directory | string | socket address of directory to query. Example: anygenre1.jamulus.io:22124. |
```

e.g. is not used elsewhere on this page. Perhaps it could be avoided here.